### PR TITLE
Remove the explicit configuration of the collector host/port within t…

### PR DESCRIPTION
**Pull request by objectiser**
…he agent, to enable the jaeger-operator to configure use of load balanced gRPC for reporting spans to the collector

Signed-off-by: Gary Brown <gary@brownuk.com>

### CODE REVIEW

**Comment by objectiser** [comment]
Related to https://issues.jboss.org/browse/MAISTRA-877

cc @pavolloffay

**Review by pavolloffay — APPROVED** [review]

File: helm/istio/charts/tracing/templates/jaeger-all-in-one.yaml [hunk 1]
```diff
@@ -25,9 +25,6 @@ spec:
       {{- end }}
 
   agent:
```
Comment (pavolloffay) [inline helm/istio/charts/tracing/templates/jaeger-all-in-one.yaml:27]: maybe remove the whole agent node

Comment (objectiser) [reply helm/istio/charts/tracing/templates/jaeger-all-in-one.yaml:27]: was wondering about that - but thought may be for consistency between the templates, just to show the component could be configured. But if others also think not relevant can remove.

**Comment by rcernich** [comment]
Could you add the jira id to the commit message please?

**Comment by objectiser** [comment]
@rcernich done

**Review by rcernich — APPROVED** [review]
lgtm

**Comment by objectiser** [comment]
@rcernich Should I create a separate PR for master/1.1.0 or will it be cherry picked?

**Comment by rcernich** [comment]
I think a separate PR with the cherry-pick, but @knrc  would know for sure.

**Review by dgn — APPROVED** [review]

**Pull request merged**